pages/services/marathon-lb/advanced/index.md

```diff
@@ -31,38 +31,72 @@ Marathon-LB has a templating feature for specifying custom HAProxy configuration
+      ssl-default-bind-options no-sslv3 no-tlsv10 no-tls-tickets
+      ssl-default-server-ciphers ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA256:ECDHE-ECDSA-AES256-SHA384:ECDHE-RSA-AES256-SHA384:DHE-RSA-AES128-SHA256:DHE-RSA-AES256-SHA256:AES128-GCM-SHA256:AES256-GCM-SHA384:AES128-SHA256:AES256-SHA256:!aNULL:!MD5:!DSS
+      ssl-default-server-options no-sslv3 no-tlsv10 no-tls-tickets
+      stats socket /var/run/haproxy/socket expose-fd listeners
+      server-state-file global
+      server-state-base /var/state/haproxy/
+      lua-load /marathon-lb/getpids.lua
+      lua-load /marathon-lb/getconfig.lua
+      lua-load /marathon-lb/getmaps.lua
+      lua-load /marathon-lb/signalmlb.lua
+    defaults
+      load-server-state-from-file global
+      log               global
+      retries                   3
+      backlog               10000
+      maxconn                3000
+      timeout connect          5s
+      timeout client          30s
+      timeout server          30s
+      timeout tunnel        3600s
+      timeout http-keep-alive  1s
+      timeout http-request    15s
+      timeout queue           30s
+      timeout tarpit          60s
+      option            dontlognull
+      option            http-server-close
+      option            redispatch
+    listen stats
+      bind 0.0.0.0:9090
+      balance
+      mode http
+      stats enable
+      monitor-uri /_haproxy_health_check
+      acl getpid path /_haproxy_getpids
+      http-request use-service lua.getpids if getpid
+      acl getvhostmap path /_haproxy_getvhostmap
+      http-request use-service lua.getvhostmap if getvhostmap
+      acl getappmap path /_haproxy_getappmap
+      http-request use-service lua.getappmap if getappmap
+      acl getconfig path /_haproxy_getconfig
+      http-request use-service lua.getconfig if getconfig
+
+      acl signalmlbhup path /_mlb_signal/hup
+      http-request use-service lua.signalmlbhup if signalmlbhup
+      acl signalmlbusr1 path /_mlb_signal/usr1
+      http-request use-service lua.signalmlbusr1 if signalmlbusr1
+    ```
+
+    In the code above, the following items have changed from the default: `maxconn`.
+
+    The current `HAPROXY_HEAD`, as well as other Marathon templates, can be [found here](https://github.com/mesosphere/marathon-lb/blob/master/Longhelp.md#haproxy_head").
 
 2.  Tar or zip the file. [Here is a handy script you can use to do this][1].
```

Comment:
Sorry, I just noticed this - not your fault,  but I think this "handy script" thing is problematic, for the following reasons:

1) it contains another copy of the old, outdated version of the template.
2) It points at an ex-employee's personal github acccount; i.e. they could delete it any time.  Also, we can't change it (we'd have to make a new one and host it somewhere)
3) It is not really all that handy, it's basic shell script stuff.  It just makes this page more complicated to maintain and follow without really helping the end-user.

I'd like to propose we just change this line to:
---------------
2.  Tar up the file: `tar czf templates.tgz templates/`
---------------
Note that the next line in this file references this script "if you use the script", I'd just delete that reference.